templates/_layouts/_footer.html

```diff
@@ -0,0 +1,23 @@
+<footer class="p-strip is-shallow" role="contentinfo">
+  <div class="row p-content__row">
+    <div class="col-12">
+      &copy; {{ now("%Y") }} Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+      <nav>
+        <ul class="p-inline-list--middot">
+          <li class="p-inline-list__item">
+            <a class="p-link--external" href="https://github.com/canonical-web-and-design/vanilla-framework/releases/latest">Vanilla framework v{{ version }}</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a class="p-link--external" href="https://www.ubuntu.com/legal">Legal info</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a class="p-link--external" href="https://github.com/canonical-web-and-design/vanilla-framework/issues/new/choose">Report a bug with this site</a>
```

Comment:
Feels like this should be "Report a bug" and in the issue templates should contain a website bug template.

Reply:
Fair point. Updated.

We already have 'Report a bug' template that seems to be generic enough to cover both the site and framework. Probably not worth making another specific one that would be mostly the same.